dev/PushNotifications/PushNotificationChannel.cpp

```diff
@@ -12,50 +12,56 @@ namespace winrt
 
 namespace winrt::Microsoft::Windows::PushNotifications::implementation
 {
+    winrt::TypedEventHandler<
+        Microsoft::Windows::PushNotifications::PushNotificationChannel,
+        Microsoft::Windows::PushNotifications::PushNotificationReceivedEventArgs> s_typedEventHandler;
+    wil::srwlock s_lock;
+
     PushNotificationChannel::PushNotificationChannel(winrt::PushNotificationChannel const& channel)
     {
         m_channel = channel;
     }
     winrt::Uri PushNotificationChannel::Uri()
     {
-        auto lock = m_lock.lock_shared();
+        auto lock = s_lock.lock_shared();
         return winrt::Uri{ m_channel.Uri() };
     }
     winrt::DateTime PushNotificationChannel::ExpirationTime()
     {
-        auto lock = m_lock.lock_shared();
+        auto lock = s_lock.lock_shared();
         return m_channel.ExpirationTime();
     }
     void PushNotificationChannel::Close()
     {
-        auto lock = m_lock.lock_exclusive();
+        auto lock = s_lock.lock_exclusive();
```

Comment:
Should this lock be shared? This looks like an operation to change the internal state of `m_channel`.